aesara/link/basic.py

```diff
@@ -231,23 +257,19 @@ def execute(*args):
             else:
                 return [variable.data for variable in outputs]
 
-        execute.thunk = thunk
-        execute.inputs = inputs
-        execute.outputs = outputs
-
         return execute
 
-    def schedule(self, fgraph: FunctionGraph) -> typing.List[Apply]:
+    def schedule(self, fgraph: FunctionGraph) -> List[Apply]:
         """Runs the scheduler (if set) or the toposort on the FunctionGraph.
 
         Parameters
         ----------
-        fgraph : FunctionGraph
+        fgraph : :py:class:`aerasa.graph.fg.FunctionGraph`
```

Comment:
Do we really need to maintain the type information separately in the docstrings?  It seems like we can get Sphinx to use the type annotations (e.g. [`sphinx-autodoc-typehints`](https://pypi.org/project/sphinx-autodoc-typehints/)).

Regardless, there's no problem with these docstring changes; I just don't want people to feel obligated to continue making them when we can use a more consistent and automated approach.